webpack.config.js/config.babel.js

```diff
@@ -55,7 +55,7 @@ export default (env: { production?: string }) => {
     };
 };
 
-const resolvePath = toResolve => path.resolve(__dirname, toResolve);
+const resolvePath = toResolve => path.resolve(__dirname, '../', toResolve);
```

Comment:
It probably doesnt matter much but this should probably be `path.resolve(__dirname, '..', toResolve)` as the `/` **might** cause problems on Windows (but as far as I know node's `path` module handles that itself).

Reply:
It should. One reason i use `path.resolve`. From the [docs](https://devdocs.io/node/path):
> Specifically, when running on a Windows operating system, the path module will assume that Windows-style paths are being used.